plugins/retina.yaml

```
@@ -0,0 +1,56 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: retina
+spec:
+  version: v0.0.2
+  homepage: retina.sh
+  shortDescription: Retina CLI allows the user to capture network traffic and metadata for the capture target, and then send the capture file to the location by Output Configuration.
```

recommended <50 chars
```suggestion
  shortDescription: Distributed network captures and telemetry
```